assets/scss/components/hfg/frontend/layout/_mobile_sidebar.scss

```diff
@@ -208,19 +157,14 @@
 	transition: all 0.3s linear;
 	visibility: hidden;
 	opacity: 0;
+	pointer-events: none;
 
 	.is-menu-sidebar & {
 		visibility: visible;
 		opacity: 1;
+		pointer-events: unset;
 	}
 }
 
 //</editor-fold>
 
-.hfg-pe {
-	pointer-events: none;
-
-	.is-menu-sidebar & {
-		pointer-events: unset;
-	}
-}
```

Comment:
This was added by @mskapusuz [here](https://github.com/Codeinwp/neve/pull/3730), and it doesn't only address the search but the menu sidebar in some contexts. I don't think we should remove this, as it might cause a regression for #3729 

References:
- https://github.com/Codeinwp/neve/issues/3729#issuecomment-1352398975
- This PR was closed due to the rollback so I think this code should not be removed - https://github.com/Codeinwp/neve/pull/3753

Reply:
I've added back the change and moved two rules to keep under the size limit. Let me know if there are other issues. Thank you!